src/addons/dragAndDrop/DraggableEventWrapper.js

```diff
@@ -135,7 +135,7 @@ class DraggableEventWrapper extends React.Component {
 /* drag sources */
 const makeEventSource = anchor => ({
   beginDrag: ({ event }) => ({ event, anchor }),
-  // canDrag: ({ event }) => event.draggable === undefined || event.draggable - e.g. support per-event dragability/sizability
+  canDrag: ({ event }) => event.draggable === undefined || event.draggable // support per-event dragability/sizability
```

Comment:
I think i'd prefer to avoid another special property on the event objects, can we use a function prop that is passed the event and returns true/false. thoughts?

Reply:
I kind of agree with this. A function that takes the event and determines if it can be dragged I think is a better option. I suppose call this `draggableAccessor`?

Reply:
Hi there,
I just moved quickly as I saw the code was already implemented.
An accessor would be a good idea but I'm afraid we'll be lost with DnD context.
https://react-dnd.github.io/react-dnd/docs-drag-source.html

For me it's easier to keep the `draggable` key in the event object for now :)

Reply:
the props to the component are passed in here, i don't think you need to mess with context at all, we just get the function the same way we get the event. Also since it's an accessor like the other sorts (it's just a function not a string or func) lets call it `isDraggable` sort of like `getNow`

Reply:
@dbuchet-bandc Yes, it's just a function that is passed to the calendar so context is not an issue, just passing it down from props should suffice.